internal/manifests/manifestutils/affinity.go

```diff
@@ -0,0 +1,37 @@
+package manifestutils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+// DefaultAffinity returns the default affinity for (most) of the Tempo components.
```

Comment:
>  for (most) of the Tempo components.

I would remove this as this is a general API

Reply:
Done